packages/wdio-mocha-framework/mocha-framework.d.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="mocha"/>
 
-declare namespace WebDriverIO {
+declare namespace WebdriverIO {
     interface Suite {
         title: string;
```

Comment:
Can you add the `file: string` property here?

Reply:
sure!